BEIMA.Client/src/pages/Devices/AddDevicePage.js

```diff
@@ -26,10 +28,32 @@ const AddDevicePage = () => {
   const [deviceImage, setDeviceImage] = useState();
   const [deviceAdditionalDocs, setAdditionalDocs] = useState();
   const [fullDeviceJSON, setFullDeviceJSON] = useState({});
+  const [deviceTypes, setDeviceTypes] = useState([]);
   
   useEffect(() => {
-    setPageName('Add Device')
-  })
+    setPageName('Add Device');
+    const loadData = async () => {
+      let types = await getDeviceTypes();
+      setDeviceTypes(types);
+    }
+   loadData()
+  },[])
+  
+  const getDeviceTypes = async () => {
+    const deviceTypeData = await GetDeviceTypeList();
+    let data = []
+    for(let i = 0; i < deviceTypeData.response.length; i++){
```

Comment:
We should use the map function! I think it would look something like this:
`let data = deviceTypeData.response.map((item) => {
return { name: item.name, id : item.id}
});`

Reply:
- [ ] Let's map this

Reply:
syntax was perfect Ashlyn